test/expectations/code_lens/minitest_with_dynamic_constant_path.exp.json

```diff
@@ -16,14 +16,15 @@
         "command": "rubyLsp.runTest",
         "arguments": [
           "/fixtures/minitest_with_dynamic_constant_path.rb",
-          "Test",
+          "<dynamic_reference>::Test",
```

Comment:
You're right it wouldn't help run the tests properly. It would just mean the IDs were unique, so if there were multiple dynamic references in the same file with similar names then the test results would be linked to the correct item in the test explorer. But that doesn't really make sense anyway, because if they have similar names they would have identical commands and run multiple tests. I think this is going to remain half working.